parachain/runtime/interlay/src/xcm_config.rs

```diff
@@ -299,10 +299,10 @@ parameter_types! {
 parameter_type_with_key! {
     // Only used for transferring parachain tokens to other parachains using DOT as fee currency. Currently we do not support this, hence return MAX.
     // See: https://github.com/open-web3-stack/open-runtime-module-library/blob/cadcc9fb10b8212f92668138fc8f83dc0c53acf5/xtokens/README.md#transfer-multiple-currencies
-    pub ParachainMinFee: |location: MultiLocation| -> u128 {
+    pub ParachainMinFee: |location: MultiLocation| -> Option<u128> {
         #[allow(clippy::match_ref_pats)] // false positive
         match (location.parents, location.first_interior()) {
-            _ => u128::MAX,
+            _ => Some(u128::MAX),
```

Comment:
```suggestion
            _ => None,
```
(also in other runtimes)